Integrations/integration-CounterTack.yml

```diff
@@ -0,0 +1,1716 @@
+description: CounterTack empowers endpoint security teams to assure endpoint protection
+  for Identifying Cyber Threats. Integrating a predictive endpoint protection platform
+configuration:
+  - display: Server URL
+    name: server
+    defaultvalue: https://democloud.countertack.com
+    type: 0
+    required: true
+  - display: User Name
+    name: credentials
+    defaultvalue: ""
+    type: 9
+    required: true
+  - display: Trust any certificate (unsecure)
+    name: insecure
+    defaultvalue: ""
+    type: 8
+    required: false
+  - display: Fetch incidents
+    name: isFetch
+    defaultvalue: ""
+    type: 8
+    required: false
+  - display: Incident type
+    name: incidentType
+    defaultvalue: ""
+    type: 13
+    required: false
+  - display: Use system proxy settings
+    name: proxy
+    defaultvalue: ""
+    type: 8
+    required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+
+    import json
+    import requests
+    import os
+    import base64
+
+    # Disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # PREREQUISITES
+    def load_server_url():
```

Comment:
there isn't a real need for this function, please use our standard way to just get rid of the trailing slash